docs/config.toml

```diff
@@ -8,5 +8,16 @@ insert_anchor_links = true
 highlight_theme = "kronuz"
 build_search_index = true
 
+[link_checker]
+# Skip link checking for external URLs that start with these prefixes
+skip_prefixes = [
+    "http://127.0.0.1",
+]
+
+# Skip anchor checking for external URLs that start with these prefixes
+# skip_anchor_prefixes = [
+#     "https://caniuse.com/",
+# ]
+
```

Comment:
what's the reason for that change?

Reply:
In fact I did a configuration documentation copy / paste.
```
[link_checker]
# Skip link checking for external URLs that start with these prefixes
skip_prefixes = [
    "http://127.0.0.1",
]

# Skip anchor checking for external URLs that start with these prefixes
# skip_anchor_prefixes = [
#     "https://caniuse.com/",
# ]
```
I wanted to avoid broken links errors with `http://127.0.0.1` (there are a several errors in the documentation), but so far I don't have to use the "anchors skip part". So I commented it out.